### DISCUSSION TITLE
Cannot type backticks with IT keyboard layout

### DISCUSSION THREAD

**Discussion by flaviocopes** [zed-industries/zed#10508] [Feature Requests]
Today backticks ` stopped working for me in Zed.

They worked before, and they work across all other apps. I type them with alt-\ with IT keyboard layout.

I remember updating to the latest stable version and creating a Zed account for the first time today, I'm not sure if some feature got enabled that prevents me from using this key combination, or if it's the update to the latest version that affected something.

Just me or someone else is having the same issue?

**Comment by flaviocopes** [comment]
Solved by unsetting a default keybinding 

```
[
  {
    "context": "Editor && !inline_completion",
    "bindings": {
      "alt-\\": null
    }
  }
]
```

(https://github.com/zed-industries/zed/blob/main/docs/src/configuring_zed__key_bindings.md#special-keyboard-layouts)

**Reply by jodaka** [reply]
This has been making me crazy! Thanks!

**Comment by mrakblt** [comment]
Why is this issue closed ? Do we have to use marked answer to fix the issue ?
